src/RemoteOwner.sol

```diff
@@ -74,20 +87,48 @@ contract RemoteOwner is ExecutorAware {
 
   /* ============ External Functions ============ */
 
-  function execute(address target, uint256 value, bytes calldata data) external returns (bytes memory) {
-    // console2.log("EXECUTE");
-    // console2.logBytes(data);
-    _checkSender();
+  function execute(address target, uint256 value, bytes calldata data) external onlyExecutorAndOriginChain onlyOwner returns (bytes memory) {
     (bool success, bytes memory returnData) = target.call{ value: value }(data);
-    // console2.log("success?", success);
-    // console2.logBytes(returnData);
-    // console2.log(abi.decode(returnData, (uint256)));
     if (!success) revert CallFailed(returnData);
     assembly {
       return (add(returnData, 0x20), mload(returnData))
     }
   }
 
+  /**
+    * @notice Renounce ownership of the contract.
+    * @dev Leaves the contract without owner. It will not be possible to call
+    * `onlyOwner` functions anymore. Can only be called by the current owner.
+    *
+    * NOTE: Renouncing ownership will leave the contract without an owner,
+    * thereby removing any functionality that is only available to the owner.
+    */
+  function renounceOwnership() external virtual onlyExecutorAndOriginChain onlyOwner {
+      _setOwner(address(0));
+  }
+
+  /**
+   * @notice Set the Owner address.
+   * @dev Can only be called once.
```

Comment:
Why can it only be set once? 
We don't check if `_pendingOwner` was already set, so I think this comment is invalid.